egs/librimix/ConvTasNet/local/prepare_data.sh

```diff
@@ -17,3 +17,9 @@ cd LibriMix
 cd $current_dir
 $python_path local/create_local_metadata.py --librimix_dir $storage_dir/Libri$n_src"Mix"
 
+
+# TODO: check folders and outdir
+#dev-clean  test-clean  train-clean-100  train-clean-360
+for split in train-360 train-100 dev-clean test-clean; do
+  $python_path local/get_text.py --libridir $storage_dir/LibriSpeech --split $split --outdir data/$split
+done
```

Comment:
The output directories don't have the `clean` in the name.. We might either want to call this 4 times without the loop, or loop over directories in the .py file.